packages/next/src/server/app-render/app-render.tsx

```diff
@@ -205,6 +205,7 @@ export const renderToHTMLOrFlight: AppPageRender = (
 
   const appUsingSizeAdjust = !!nextFontManifest?.appUsingSizeAdjust
 
+  // TODO: fix this typescript
```

Comment:
This is an artifact of having `RenderOpts` be declared with partial properties so that it can mesh with the other versions of `RenderOpts`. Flagging here as a TODO.